flight/mcu/lib/teensyShared/BarometerSensor.h

```diff
@@ -1,20 +1,21 @@
-#ifndef BAROMETERSENSOR_H
-#define BAROMETERSENSOR_H
+#pragma once
 
-#include <MS5611.h>
+#include "MS5611.h"
+#include "pins.h"
 
-class BarometerSensor {
+struct BarometerSensor {
    public:
-    void readReadings();
-    float getPressure();
-    float getTemperature();
-    float getAltitude();
+    MUTEX_DECL(mutex);
+
+    void init();
+    void refresh();
+    float getPressure() const;
+    float getTemperature() const;
+    float getAltitude() const;
 
    private:
-    MS5611* MS;
-    float pressure;
-    float temperature;
-    float altitude;
+    MS5611 MS{MS5611_CS};
```

Comment:
For documentation purposes, can you comment here describing why storing the sensor object itself is as good or better than storing the pointer to its object?